src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,88 @@
+package duke;
+
+import duke.storage.Storage;
+import duke.exceptions.DukeException;
+import duke.handlers.*;
+import duke.models.TaskList;
+import duke.services.Parser;
+import duke.utils.Commands;
+
+import java.util.Scanner;
+import static duke.services.Ui.dukePrint;
+
+public class Duke {
+    private Storage storage;
+    private final Parser parser;
+    private TaskList taskList;
+
+    /**
+     * Constructor for a Duke instance.
+     */
+    public Duke() {
+        parser = new Parser();
+        storage = new Storage();
+        taskList = new TaskList();
+    }
+
+    /**
+     * Run method to execute the Duke instance.
```

Comment:
If you see the general format of javadocs for method, they usually write in present tense and verb 
ie. Executes the Duke instance